src/java/org/apache/cassandra/cql3/CQL3Type.java

```diff
@@ -34,7 +34,6 @@
 import org.apache.cassandra.schema.Types;
 import org.apache.cassandra.serializers.CollectionSerializer;
 import org.apache.cassandra.serializers.MarshalException;
-import org.apache.cassandra.transport.ProtocolVersion;
```

Comment:
Do we really want to lose the ability to have version specific encodings?  Such that if we want it again because we change someone in 5.1 someone has to go and undo all these changes?

Reply:
Its dead code, easy to add back if we ever have such a use case